src/main/main.go

```diff
@@ -33,15 +34,25 @@ func main() {
 	}
 
 	krakenAPI := krakenapi.New(args.Key, args.Secret)
-	var notifierInstance *notifier.Notifier
-	if args.CredentialsFile != "" {
-		var gmailer notifier.Notifier = notifier.MustNewGMailer(args.CredentialsFile, "me")
-		notifierInstance = &gmailer
+	var notifiersInstance []*notifier.Notifier
```

Comment:
I guess we can call this var just `notifiers` now, I had to name it `notifierInstance` to not conflict with the package name